blockware/matrix/lib/readme.txt

```diff
@@ -0,0 +1,36 @@
+
+This directory is intended for the project specific (private) libraries.
```

Comment:
Fine to remove this

Reply:
agree i copied it when i copied fireworks maybe worthy a pass to create an example that keeps this and remove it everywhere else?